scripts/e2e_test/start_test.sh

```diff
@@ -250,7 +250,7 @@ client_test() {
 api_test() {
   pushd ../apitest/pytest
   python3 -m pip install -r requirements.txt
-  pytest --host ${CONTROLLER_HOST} --port 80
+  pytest --host ${CONTROLLER_HOST} --port 80 || exit 1
```

Comment:
make the `set -e` as default (remove the if statement)?
https://github.com/star-whale/starwhale/blob/cba5c49ff1f7dec9e5e70f2484b29841a9b89509/scripts/e2e_test/start_test.sh#L5-L7